packages/strapi-plugin-content-manager/utils/wrap-bad-request.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+module.exports = fn => async () => {
+  try {
+    await fn();
+  } catch (error) {
+    if (strapi.errors.isBoom(error)) {
+      throw error;
+    }
+
+    // these are errors like unique constraints
+    strapi.log.error(error);
+    throw strapi.errors.badRequest('Invalid input data. Please verify unique constraints');
```

Comment:
There are no notion of 500 errors here just exceptions. It will just return this error in case of a bigger error instead of returning a 500. I just kept the previous behavior here

Reply:
I meant unexpected errors that would result in sending the wrong error message. Are we sure the error will always be because of a unique constraints?
Ok if that was the previous behavior, won't fix it now

Reply:
Actually we don't know for sure it is but that's either the unique or an unknow error (unknow for us too like db connecttion or something). I can remove the unique constraint part but I figure it would help in those cases and the rest of the time the dev will have to go and check the log